### ISSUE TITLE
Unable to zoom website

### ISSUE THREAD

**Issue by ZoeBijl** [nikkilr88/nikkilr88.github.io#4]
The `user-scalable=no` in the viewport meta information causes accessibility issues. Some people need to increase the font size or zoom in to get a better look at what they’re reading/viewing. This is related to [WCAG 2.1 §1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow). That basically says that your stuff needs to be able to scale to 400%.

This got so bad that Apple started to ignore `user-scalable` in iOS 10:

> Zooming Everywhere
> 
> Safari on iOS 10 allows the user to pinch zoom on every page. As a developer, you should be aware of this, and make sure your content works well when zoomed.
> 
> What changed? Prior to iOS 10, Safari allowed the content to block the user from zooming on a page by setting `user-scalable=no` in the viewport, or appropriate `min-scale` and `max-scale` values. This unfortunately enabled pages to pick a text size that was unreadable while giving the user no way to zoom. Also, there is now such a wide range of devices with different display dimensions, screen resolutions, pixel densities… it is very difficult to choose an appropriate text size in a design.

Source: [New Interaction Behaviors in iOS 10](https://webkit.org/blog/7367/new-interaction-behaviors-in-ios-10/)

https://github.com/nikkilr88/nikkilr88.github.io/blob/aa8185195ac758c46161c082e536c26d9fa482dd/index.html#L8

Suggest `user-scalable=no` is removed ⭐️.

Out of interest: what issue did you want to solve with `user-scalable=no`?
